tests/scripts/recreate-test-infrastructure.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -uxe
+
+
+URL=$(oc config view --minify -o jsonpath='{.clusters[*].cluster.server}')
+
+if [ ${URL} != "https://127.0.0.1:8443" ]; then
+    echo "You are not in a local cluster. Stopping now!!!"
+fi
+
+if ! oc get clusterroles | grep request_role; then
+echo "{ \"apiVersion\": \"v1\", \"kind\": \"ClusterRole\", \"metadata\": { \"name\": \"request_role\" }, \"rules\": [ { \"verbs\": [\"create\"], \"resources\": [\"projectrequests\"] }, { \"verbs\": [\"get\",\"list\",\"watch\"], \"resources\": [\"*\"] } ] }" \
+    | oc create -f -
+fi
+
+if [ ! -d ${BASH_SOURCE%/*}/../../ods-config ]; then
+    mkdir -p ${BASH_SOURCE%/*}/../../ods-config
+fi
+
+${BASH_SOURCE%/*}/create-env-from-local-cluster.sh --base-oc-dir=${HOME}/openshift.local.clusterup --output ${BASH_SOURCE%/*}/../../ods-config/ods-core.env
+
+if [ -d "${HOME}/ods-configuration" ]; then 
+    rm -rf ${HOME}/ods-configuration
+fi
+
+NAMSPACE="cd"
+REF="ci/cd"
+
+if ! oc whoami; then
+    echo "You must be logged in to the OC Cluster"
+fi
+
+if oc project ${NAMSPACE}; then
+    oc delete project ${NAMSPACE}
+fi
+
+${BASH_SOURCE%/*}/../../ods-setup/setup-ods-project.sh --verbose --force --namespace ${NAMSPACE}
+
+${BASH_SOURCE%/*}/deploy-mocks.sh  --verbose
+sleep 10 # Waiting for service to boot up
```

Comment:
Hmm. Should this rather be in `deploy-mocks`? Maybe as a param `--wait`?

Reply:
Added --wait